README.md

```diff
@@ -27,6 +27,16 @@ Run `cargo build` or `cargo build --release` to build in release mode.
 ## Test
 Run `cargo test` for unit tests.
 
+## Binaries
+The following binaries are provided (use `cargo install` to install them):
+- **parquet-schema** for printing Parquet file schema and metadata.
+`Usage: parquet-schema <file-path> [verbose]`, where `file-path` is a path to the Parquet file,
+and optional `verbose` allows to print full metadata.
+
+- **parquet-read** for reading records from a Parquet file.
+`Usage: parquet-read <file-path> [num-records]`, where `file-path` is a path to the Parquet file,
+and optional `num-records` is a number of records to read from a file.
```

Comment:
nit:  `is a number` -> `is the number`. Also perhaps point out that when `num-records` is not specified, all records will be printed.

Reply:
Yes, will do. Thanks.